src/js/tracks/text-track-settings.js

```diff
@@ -321,17 +323,12 @@ class TextTrackSettings extends Component {
    * @method restoreSettings
    */
   restoreSettings() {
-    let err;
     let values;
 
     try {
-      [err, values] = safeParseTuple(window.localStorage.getItem('vjs-text-track-settings'));
```

Comment:
It seems like overkill to have two `try` blocks (even if one is hidden in a dependency). This way we only have one.